backend/run.sh

```diff
@@ -19,6 +20,11 @@ source tools/seed_cog_baseline.sh
 setup_env
 gonogo "setup_env"
 
+#####
```

Comment:
Move to the end/late in the boot sequence. After the whole API standup/migrate/teardown dance.

Reply:
NEVERMIND. Leave this early in the sequence.

WHY?

What if we crashed, and the DB is in `enable` mode? We could enter the `migration` stage, run a big migration, and rewrite every row of `singleauditchecklist` That would all be documented as a curation, when it kinda isn't. (Eg. if we add a column with a new default value.) 

So, we should make sure it is disabled *before* the middle sequence, not after.